loopin.go

```diff
@@ -401,7 +418,7 @@ func (s *loopInSwap) publishOnChainHtlc(ctx context.Context) (bool, error) {
 	s.log.Infof("Publishing on chain HTLC with fee rate %v", feeRate)
 	tx, err := s.lnd.WalletKit.SendOutputs(ctx,
 		[]*wire.TxOut{{
-			PkScript: s.htlc.PkScript,
+			PkScript: s.htlcP2WSH.PkScript,
```

Comment:
If possible it would be helpful to isolate the change of internal publication to p2wsh in a separate commit.

Reply:
done